### DISCUSSION TITLE
Can you convert Word documents with comments in a PDF?

### DISCUSSION THREAD

**Discussion by edmound-hubbard-enverus** [gotenberg/gotenberg#675] [Q&A] [answered]
Didn't see much in documentation around this.  We have a Word document which contains comments.  When you export to a PDF in Word the comments are visible, is this possible with using LibreOffice?  Is there a parameter we need to pass in to include comments?

**Comment by gulien** [comment, marked as answer]
Hello @edmound-hubbard-enverus 

No idea if that’s possible with LibreOffice, but it’s definitely not doable with Gotenberg at the moment.

Feel free to open an issue so that I have it on my TODO list, if feasible.

**Reply by edmound-hubbard-enverus** [reply]
I had a hunch that was the case.  I'll dig a little more to see what, if any the command would be.  Their documents talk about using the UI interface and enabling comments as PDF annotations.

Thanks for the quick response!
